src/components/Layer2/Layer2Onboard.tsx

```diff
@@ -351,11 +355,11 @@ const Layer2Onboard: React.FC<IProps> = ({
         )}
         {/* EthLogo */}
         <Box {...gridContentPlacementStyles.logo}>
-          <GatsbyImage
-            image={ethIcon}
+          <Image
+            src={ethIcon}
             objectFit="contain"
             alt={ethIconAlt}
-            w="full"
+            w="50px"
```

Comment:
w' is a chakra style prop that only chakra understands. In this case, we should use `width` + an integer to tell Next that we want to display a 50px width image, so it will resize it.

```suggestion
            width={50}
```

For example, you can see here that if we don't specify the `width`, the browser would load the original image (it could be a pretty heavy image) and it will only be visually resized by the `w` prop.
![image](https://github.com/ethereum/ethereum-org-fork/assets/468158/06ba1243-37f0-411c-9b80-9d0230ea6c14)